assets/sass/_navigation.scss

```diff
@@ -91,7 +91,7 @@ Style guide: Navigation.Primary Navigation
     &.active,
     &.is-active {
       border-color: $border-contrast-colour;
-      background-color: $background-contrast-colour;
+      background-color: $lighter-aqua;
```

Comment:
We should tie our colours to their function; the first section in the `colours.scss` partial lists our colours according to their name, and then we redefine them to their function.

Just nitpicking otherwise :S